sw/ext/tlsf/readme.txt

```diff
@@ -0,0 +1 @@
+see http://tlsf.baisoku.org
```

Comment:
also, when we did this, the code was not yet published on GitHub, only tarballs and maybe SVN...